roles/openshift_metrics/tasks/generate_rolebindings.yaml

```diff
@@ -46,3 +46,75 @@
     files:
     - "{{ mktemp.stdout }}/templates/hawkular-cluster-role.yaml"
     delete_after: true
+
+- name: generate the metrics-server cluster role
+  template:
+    src: metrics-server-role.js2
+    dest: "{{ mktemp.stdout }}/templates/metrics-server-role.yaml"
+  changed_when: no
+
+- name: generate auth-delegator role binding for the metrics-server service account
+  template:
+    src: metrics-server-rolebinding.j2
+    dest: "{{ mktemp.stdout }}/templates/metrics-server-auth-delegator-rolebinding.yaml"
+  vars:
+    cluster: True
+    obj_name: metrics-server:system:auth-delegator
+    labels:
+      kubernetes.io/cluster-service: |
```

Comment:
what's up with the yaml block notation?

just do `kubernetes.io/cluster-service: "true"`

Reply:
It strips the quotes then the oc apply barfs with "bad json" for (unquoted) true.

Reply:
yeah, put it in the raw template.  Doing it in the ansible is being needlessly complicated here.